scripts/SasNetworkGuard.psm1

```
@@ -0,0 +1,53 @@
+
+    begin { $lines = New-Object System.Collections.Generic.List[string] }
+    process {
+        if ($null -ne $Text) {
+            foreach ($line in ($Text -split "`r?`n")) { [void]$lines.Add($line) }
+        }
+    }
+    end {
+        foreach ($line in $lines) {
+            if ($line -match '^\s*SSID\s*:\s*(.*)\s*$') {
+                $ssid = $Matches[1].Trim()
+                if ([string]::IsNullOrWhiteSpace($ssid)) { return 'unknown' }
```

PSScriptAnalyzer (1.25.0)

[info] The cmdlet 'Get-SasCurrentWifiSsidFromNetshText' returns an object of type 'System.String' but this type is not declared in the OutputType attribute.

(PSUseOutputTypeCorrectly)